toolkit/data-sources/db-sync/testdata/governed-map/migrations/2_data.sql

```diff
@@ -72,7 +79,8 @@ VALUES             (0 , bhash_0, NULL    , 189410  , NULL         , NULL    , NU
 INSERT INTO tx ( id            , hash            , block_id, block_index, out_sum, fee, deposit, size, invalid_before, invalid_hereafter, valid_contract, script_size )
     VALUES     ( consumed_tx_id, consumed_tx_hash, 1       , 0          , 0      , 0  , 0      , 1024, NULL          , NULL             , TRUE          , 1024        )
               ,( ins_tx_id     , hash1           , 1       , 1          , 0      , 0  , 0      , 1024, NULL          , NULL             , TRUE          , 1024        )
-              ,( del_tx_id     , hash2           , 4       , 0          , 0      , 0  , 0      , 1024, NULL          , NULL             , TRUE          , 1024        )
+              ,( ins_tx_ida    , hash2           , 1       , 2          , 0      , 0  , 0      , 1024, NULL          , NULL             , TRUE          , 1024        )
```

Comment:
Transaction that inserts "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" is after transaction that inserts "22222222222222222222222222222222", but the test case for block 1 still asserts on "22222222222222222222222222222222".